testing/src/test/java/io/stargate/it/http/RestApiv2Test.java

```diff
@@ -364,6 +366,150 @@ public void deleteTable() throws IOException {
         HttpStatus.SC_NO_CONTENT);
   }
 
+  @Test
+  public void createIndex() throws IOException {
```

Comment:
You can also test the different types inside `io.stargate.it.http.RestApiv2DseTest`. In that test you can create and index with SAI.

Reply:
Added a test to `io.stargate.it.http.RestApiv2DseTest`, please see if it's right.